src/catalog/category-permissions.md

```diff
@@ -36,7 +36,7 @@ When enabled, a new “Category Permissions” section appears on the Category p
 
     - Set **Allow Browsing Category** to `Specified Customer Groups`.
 
-    - In the **Customer Groups** box, select each group that is allowed to browse through products in the category. (For multiple options, hold down the Ctrl key, and click each group.)
+    - In the **Customer Groups** box, select each group that is allowed to browse through products in the category. (To select multiple groups, hold down the Ctrl key (PC) or the Command key (Mac) as you click each group.)
```

Comment:
seems like you could remove the ( and just have the statement

Reply:
I could go either way.